package.json

```
@@ -4,11 +4,13 @@
   "description": "Type definitions for Corvid by Wix",
   "scripts": {
     "lint": "eslint .",
-    "build": ":",
+    "build": "npm run generate-dts",
     "test": "jest",
-    "generate-dts": "docworks dts -r git@github.com:wix-private/wix-code-docs.git -o declaration -d temp_declarations --wixselector && npm run copy-declarations temp_declarations",
-    "postpublish" : "PACKAGE_VERSION=$(cat package.json | grep \\\"version\\\" | head -1 | awk -F: '{ print $2 }' | sed 's/[\",]//g' | tr -d '[[:space:]]') && git tag v$PACKAGE_VERSION && git push --tags",
-    "copy-declarations": "scripts/copy_declarations.sh"
+    "clone-repo": "rm -rf corvid_docs && git clone git@github.com:wix-private/wix-code-docs.git corvid_docs",
+    "generate-dts": "npm run clone-repo && npm run generate-dts:declaration && npm run generate-dts:selector && rm -rf corvid_docs",
```

Consider making `corvid_docs` a parameter to all other scripts, so you only have to write it in this one line. For example:
```suggestion
    "generate-dts": "npm run clone-repo -- corvid_docs && npm run generate-dts:declaration -- corvid_docs && npm run generate-dts:selector -- corvid_docs && rm -rf corvid_docs",
```

This will require making some minor changes in the following places:
https://github.com/wix-incubator/corvid-types/blob/dae32d20ae425932558e99ebc15bf64c6bf766e9/package.json#L9
https://github.com/wix-incubator/corvid-types/blob/dae32d20ae425932558e99ebc15bf64c6bf766e9/package.json#L11
https://github.com/wix-incubator/corvid-types/blob/dae32d20ae425932558e99ebc15bf64c6bf766e9/scripts/selector-declaration-generator.js#L6


Alternatively (and perhaps a nice approach) is to create one `generate-dts.js` script that will orchestrate it all 🎼 🎻

---

So we go with the alternative and nicer solution, and created one generated file with all constants and other stuff :)